.github/workflows/test.yml

```diff
@@ -7,59 +7,158 @@ on:
       - master
   # And all pull requests
   pull_request:
+  schedule:
+    # Scheduled run over at 0217 UTC Sunday to detect any upstream breaks.
+    # * is a special character in YAML so you have to quote this string
+    - cron:  '17 2 * * 0'
 
   # Allows you to run this workflow manually from the Actions tab
   workflow_dispatch:
 
 concurrency:
   # Cancels jobs running if new commits are pushed
-  group: ${{ github.workflow }}-${{ github.event.pull_request.number || github.ref }}
+  group: >
+    ${{ github.workflow }}-
+    ${{ github.event.pull_request.number || github.ref }}
   cancel-in-progress: true
 
 jobs:
   build:
     name: "Build and test asQ"
-    # The type of runner that the job will run on
-    # runs-on: ubuntu-latest
+    # Use Firedrake's Linux runners
     runs-on: [self-hosted, Linux]
-    # The docker container to use.
+    # Use the firedrake container without asQ installed - we want to run the repo version.
     container:
-      image: firedrakeproject/firedrake-vanilla:latest
+      image: firedrakeproject/firedrake-vanilla-default:2025.4.1
```

Comment:
is there a way to automatically warn us if there has been a subsequent update on the release?

Reply:
Possibly we could set up a notification from the firedrake repo or Dockerhub. I think really we want separate workflows for release and master, but we can't do that until this is released and asQ/master is brought up to date with firedrake/master.

I was being cautious here, possibly overly so, to make sure the right container was used before we make a matching release. I'd be fine switching back to `latest`.